src/subscriptions/ManageSubscriptionsPage.test.jsx

```diff
@@ -31,33 +32,19 @@ describe('<ManageSubscriptions />', () => {
       };
 
       jest.useFakeTimers();
```

Comment:
Any specific reason to use `useFakeTimers` here?

Reply:
it was done so this use effect could be tested
```
useEffect(() => {
    if (stripeCustomerPortalURL) {
      window.location.href = stripeCustomerPortalURL;
    }
  }, [stripeCustomerPortalURL]);
  ```
and we could run the timers and  expect mockHrefSetter toHaveBeenCalledWith the expected url